mlir/tools/mlir-miopen-lib/CMakeLists.txt

```diff
@@ -44,6 +44,11 @@ llvm_update_compile_flags(mlir-miopen-lib-test)
 target_link_libraries(mlir-miopen-lib-test PRIVATE MLIRMIOpenThin ${LIBS})
 mlir_check_link_libraries(mlir-miopen-lib-test)
 
+install(TARGETS MLIRMIOpenThin
+  DESTINATION lib)
```

Comment:
This needs to be `libMLIRMIOpen` for the static library export, and needs to be guarded by `NOT BUILD_SHARED_LIBS`